scripts/uberenv/packages/lua/package.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
```

Comment:
I'm assuming that spack's built-in `lua` had issues with axom's spack config?

Reply:
They jam in luarocks into their lua package which is how you get lua modules.  I have spoken to them about adding a regular lua package but haven't got much traction. I limited what we were building and will test later if we can work with the extra stuff they are adding later.